requirements/common.txt

```diff
@@ -4,7 +4,7 @@ Flask-Script>=2.0.5,<2.1
 Flask-SQLAlchemy>=2.1,<2.2
 Flask-Migrate>=2.0.0,<3
 Flask-Login>=0.4,<1
-Flask_Dance>=1.3.0
+Flask-Dance>=1.3.0
```

Comment:
See the format in which other dependencies are pinned

Reply:
Will resolve this. Just checking for the main reason for Travis fail.